packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -93,8 +140,7 @@ class WidgetFlutterBinding extends FlutterBinding {
 
 /// Inflate the given widget and attach it to the screen.
 void runApp(Widget app) {
-  WidgetFlutterBinding.ensureInitialized();
-  WidgetFlutterBinding.instance._runApp(app);
+  new WidgetFlutterBinding()._runApp(app);
```

Comment:
Since there's now several binding mixin classes, each of which with its own instance getter, you have to explicitly instantiate the concrete class you want to use as the amalgamation of all the bindings, otherwise it's not obvious what to instantiate when you call the instance getters.

For WidgetFlutterBinding, since it's concrete, we could have a lazily initialized static final instead of a constructor, but then it's a bit magic -- this instance getter autoinitialises, but the others don't. The explicit constructor call is clearer, I think.